hivemq-edge/src/frontend/src/locales/en/translation.json

```
@@ -1204,6 +1211,11 @@
         "title": "Update the combiner",
         "success": "We've successfully updated the combiner for you.",
         "error": "There was a problem trying to update the combiner"
+      },
+      "delete": {
+        "title": "Delete the combiner",
```

non-blocking:

```suggestion
        "title": "Combiner deleted",
```
Imho past tense applies sounds better for the toast headers. I see you've got the same pattern for update as well.

---

Yes, see what you mean.
Can't do it now, I've been reusing too much to get the proper structure :-)